test/unit/context.test.js

```diff
@@ -35,8 +36,16 @@ test.afterEach((t) => {
 })
 
 test('locates correct config file path', (t) => {
+  const configPath = getConfigPath()
+  t.is(configPath, resolve(sync('.contentfulrc.json')))
```

Comment:
U have a `.contentfulrc.json` on your disc or even your home directory. Fresh installs and the CLI probably won't have that. You may want to add a `.contentfulrc.json` file to the repository root directory while the tests are running. You gonna need this for e2e tests, too.